treeherder/changelog/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+
+
+class Changelog(models.Model):
+    id = models.CharField(max_length=255, primary_key=True)
```

Comment:
We typically use `id = models.AutoField(primary_key=True)` or BigAutoField if necessary.